app/wizards/schools/register_ect_wizard/registration_store/queries.rb

```diff
@@ -26,6 +26,17 @@ def contract_start_date
           @contract_start_date ||= ContractPeriod.containing_date(registration_store.start_date&.to_date)
         end
 
+        def lead_provider_partnership_for_contract_period(school:)
```

Comment:
Possibly rename this to be partnerships? Just housekeeping - I realise you were refactoring in this PR!